src/openrct2-ui/windows/RideConstruction.cpp

```diff
@@ -2546,6 +2546,8 @@ void sub_6C94D8()
         _currentTrackSelectionFlags &= ~TRACK_SELECTION_FLAG_RECHECK;
     }
 
+    uint32_t curTime;
```

Comment:
Initialize the variable here

Reply:
I wanted to avoid calling `platform_get_ticks()` in construction states where it isn't relevant. I've added scope to the case blocks instead so we don't have to declare it before the switch statement.